webapp/graphite/remote_storage.py

```diff
@@ -267,3 +284,31 @@ def connect(self):
       break
     if not self.sock:
       raise socket.error(msg)
+
+# support for HTTPS connections with timeout for < python 2.6
+class HTTPSConnectionWithTimeout(httplib.HTTPSConnection):
```

Comment:
Is support for python < 2.6 still required for HTTP? I would happily remove the connector_class_selector along HTTPSConnectionWithTimeout and HTTPConnectionWithTimeout